packages/teams-js/src/internal/uuidObject.ts

```diff
@@ -4,14 +4,29 @@ import { generateGUID, validateUuid } from './utils';
  * @internal
  * Limited to Microsoft-internal use
  *
- * UUID object
+ * Represents a UUID (Universally Unique Identifier) object.
+ * This class provides a way to generate, validate, and represent UUIDs as strings.
  */
 export class UUID {
+  /**
+   * Creates an instance of the UUID class.
+   * If no UUID string is provided, a new UUID is generated.
+   *
+   * @param {string} [uuid=generateGUID()] - The UUID string. Defaults to a newly generated UUID.
```

Comment:
>{string} [uuid=generateGUID()] [](http://example.com/codeflow?start=12&length=30)

Can you say more about this typedoc syntax? I haven't seen it elsewhere in this codebase and I'm not totally sure what it does. I generated the docs from your change I see this:


...I don't know what those docs will look like when generated by the Microsoft doc system, but it seems like if your intention was to highlight the default value when not specified, it's not coming through in the docs.

Reply:
I will look into this after merging this PR. We don't generate docs for the private folder, so this shouldn't have any public documentation-facing impact.